blockstack/lib/nameset/namedb.py

```diff
@@ -1120,7 +1125,75 @@ def get_num_names_in_namespace( self, namespace_id ):
         cur = self.db.cursor()
         return namedb_get_num_names_in_namespace( cur, namespace_id, self.lastblock )
     
+    def get_num_names_in_namespace_at_block( self, namespace_id, block_id ):
```

Comment:
This only counts on-chain names, but I think that's okay since we're only looking for on-chain names in `.miner`?

Reply:
Yes that is correct